app/commands.py

```diff
@@ -314,36 +315,36 @@ async def maplink(ctx: Context) -> str | None:
 async def recent(ctx: Context) -> str | None:
     """Show information about a player's most recent score."""
     if ctx.args:
-        target = app.state.sessions.players.get(name=" ".join(ctx.args))
+        target = await users_repo.fetch_one(name=ctx.args[1])
```

Comment:
I think we can keep this as `" ".join(ctx.args)`, that way it can support spaces in usernames

e.g. `!recent frosti drinks`